app/frontend/components/domains/step-code/index.tsx

```diff
@@ -70,17 +71,20 @@ export const StepCodeForm = observer(function NewStepCodeForm() {
           }
         >
           {isLoaded && currentPermitApplication && (
-            <Container my={10} maxW="780px" px={0}>
-              {!currentStepCode ? (
-                <VStack spacing={8} align="start" w="full" pb={20}>
-                  <Title />
-                  <Info />
-                  <DrawingsWarning />
-                  <H2KImport />
-                </VStack>
-              ) : (
-                <StepCodeChecklistForm />
-              )}
+            <Container maxW="container.lg">
+              <BuilderTopFloatingButtons top="24" />
```

Comment:
As per @karenkun , move this component to `shared` screen and rename as it's more generally used now

Reply:
yupp good call - throught briefly about this but got lazy 😅